### PR TITLE
Stylingcont

### PR DESCRIPTION
**Pull request by majakudlicka**
More styling - basic styling more or less done for the app

### CODE REVIEW

**Comment by des-des** [comment]
@majakudlicka You seem to have been pushing lots since you assigned me. Not sure if you are still working on this.

Can you assign me when you are sure that you are not doing more work on this branch.

**Comment by majakudlicka** [comment]
Yes sorry I pushed than realised something was broken on heroku. I should have closed this pull request then re-open. 
Anyway, I am finished working on this branch.

File: react-ui/public/style.css [hunk 1]
```diff
@@ -52,16 +75,99 @@ html * {
 }
 
 .welcome {
+<<<<<<< HEAD
```
Comment (des-des) [inline react-ui/public/style.css]: ops

File: react-ui/public/style.css [hunk 2]
```diff
@@ -52,16 +75,99 @@ html * {
...
+  margin-bottom:5%;
+  background-color: #dbebc0;
+=======
```
Comment (des-des) [inline react-ui/public/style.css]: ..

File: react-ui/src/actions/job_apply.js [hunk 3]
```diff
@@ -1,36 +1,21 @@
...
-            });
-        });
-    
+
     axios
-        .get('/api/apply')
+        .put('/api/apply', {jobId: jobId})
+      
         .then(response => {
+          console.log('inside actions ', jobId)
```
Comment (des-des) [inline react-ui/src/actions/job_apply.js]: No

File: server/index.js [hunk 4]
```diff
@@ -219,16 +219,20 @@ server.register(plugins, err => {
             var subject = 'New job application';
             var text =
                 'Someone has applied for the job you posted. Go to your profile to find out more.';
-            data.submitApplication(request.payload.job_id, (err, res) => {
+            data.submitApplication(request.payload.jobId, (err, res) => {
+                console.log('inside put route', request.payload.jobId);
```
Comment (des-des) [inline server/index.js]: :fire:

File: server/index.js [hunk 5]
```diff
@@ -219,16 +219,20 @@ server.register(plugins, err => {
...
                 if (err) {
+
                     reply(
                         Boom.serverUnavailable(
                             'Failed to retrieve data from database'
                         )
                     );
                 } else {
+                    console.log('inside email route');
```
Comment (des-des) [inline server/index.js]: :fire:

File: server/index.js [hunk 6]
```diff
@@ -219,16 +219,20 @@ server.register(plugins, err => {
...
                     sendEmail(from, to, subject, text, (err, res) => {
                         if (err) {
+                          console.log()
```
Comment (des-des) [inline server/index.js]: :fire:

File: react-ui/public/style.css [hunk 7]
```diff
@@ -52,16 +75,99 @@ html * {
...
+    justify-content: center;
+    align-items: center;
+}
+=======
     .container-flex{
         display: flex;
         justify-content: center;
         align-items: center;
     }
+>>>>>>> master
```
Comment (des-des) [inline react-ui/public/style.css]: :fire:

File: react-ui/public/style.css [hunk 8]
```diff
@@ -52,16 +75,99 @@ html * {
...
+    align-items: center;
+}
+
+.parent-container {
+    height: 80vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+=======
```
Comment (des-des) [inline react-ui/public/style.css]: :fire:

File: react-ui/public/style.css [hunk 9]
```diff
@@ -52,16 +75,99 @@ html * {
 }
 
 .welcome {
+<<<<<<< HEAD
+  margin-bottom:5%;
+  background-color: #dbebc0;
+=======
     margin-bottom:5%;
     background-color: #F8F8F8;
+>>>>>>> master
```
Comment (des-des) [inline react-ui/public/style.css]: :fire:

File: react-ui/public/style.css [hunk 10]
```diff
@@ -52,16 +75,99 @@ html * {
...
 }
 
-@media (min-width: 576px) {
 
+<<<<<<< HEAD
```
Comment (des-des) [inline react-ui/public/style.css]: :fire:

File: react-ui/src/components/pages/MyJobs.js [hunk 11]
```diff
@@ -70,22 +70,27 @@ class MyJobs extends Component {
         );
     }
 
+    // let myJobsList = myJobs && myJobs.myJobsList;
```
Comment (des-des) [inline react-ui/src/components/pages/MyJobs.js]: :fire:

File: react-ui/src/components/pages/RegisterStudent.js [hunk 12]
```diff
@@ -7,15 +7,24 @@ import {connect} from 'react-redux';
...
+                    <div className="col-md-4 col-md-offset-4 ">
+                        <h3>Student Registration</h3>
+
+                        <p>
+                            <i>Fields marked with * are mandatory</i>
+                        </p>
+
+                        <Form_Register_Student
+                            btnText={'Sign Up'}
+                            //student={this.props.student}
```
Comment (des-des) [inline react-ui/src/components/pages/RegisterStudent.js]: :fire:

File: react-ui/src/components/pages/SearchJobs.js [hunk 13]
```diff
@@ -31,7 +31,7 @@ class BrowseJobs extends Component {
         if (desc.length <= 100) {
             return desc;
         } else {
-            return desc.slice(0, 100) + '...';
+            return desc.slice(0, 220) + '...';
```
Comment (des-des) [inline react-ui/src/components/pages/SearchJobs.js]: Move this 220 into a variable. See how on line 31 you have forgotten to update the value ..

**Comment by majakudlicka** [comment]
All done, sorry for all the leftover console logs

**Pull request merged**